Task7/vertex-ai-qwikstart/requirements.txt

```diff
@@ -0,0 +1,4 @@
+explainable-ai-sdk==1.3.0
+tensorflow-io==0.15.0
+pyarrow
+
```

Comment:
Rename requirements.txt and requirements2.txt to something meaningful or explain when to use